src/oogeso/core/simulator.py

```diff
@@ -54,30 +56,56 @@ def __init__(self, data: EnergySystemData):
 
     def runSimulation(
         self,
-        solver,
-        timerange=None,
-        timelimit=None,
-        store_duals=None,
-        write_yaml=False,
-    ):
-        """Solve problem over many timesteps - rolling horizon
+        solver: str,
+        timerange: Tuple[int, int] = None,
+        timelimit: int = None,
+        return_variables: Sequence[str] = None,
+        store_duals: dict = None,
+        write_yaml: bool = False,
```

Comment:
Are these optional since they default to None?

Reply:
Yes, optional. Have added Optional[...]